packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -111,6 +117,19 @@ function isContextBeingChecked(contextName: string, experimental: boolean): bool
   return !!$kubernetesContextsCheckingStateDelayed?.get(contextName);
 }
 
+function getResourcesCount(
+  contextName: string,
+  resourceName: SelectedResourceName,
+  experimental: boolean,
+): number | undefined {
+  if (experimental) {
+    return $kubernetesResourcesCount.find(
+      resourcesCount => resourcesCount.contextName === contextName && resourcesCount.resourceName === resourceName,
+    )?.count;
+  }
+  return $kubernetesContextsState.get(contextName)?.resources[resourceName];
+}
```

Comment:
question: should it provide undefined or 0 even if not found ?

for example UI would display 'undefined' ?

Reply:
Yes, I'm not sure either. The value is undefined when the user doesn't have access to the resource. The design will be given in https://github.com/podman-desktop/podman-desktop/issues/10385, I would prefer to have the design before to decide

Reply:
I changed, to protect displaying `undefined` and display nothing instead